test/integration/tests/button/props.js

```diff
@@ -135,4 +135,33 @@ describe(`paypal button component props`, () => {
             });
         }));
     });
+
+    it('should render a button and get the renderedButtons props', () => {
+        return ZalgoPromise.try(() => {
+            return wrapPromise(({ expect, avoid }) => {
+                let onRender = ({ xprops }) => {
+                    const renderedButtons = xprops.renderedButtons;
+                    if (!renderedButtons.length) {
```

Comment:
Can we check for the exact array values here? It would be great to confirm that all the expected funding sources show up.

Reply:
Yes! I'll add this logic

Reply:
Done